test/python/expected/pyodbc/ddl_views.out

```diff
@@ -0,0 +1,36 @@
+SET ANSI_NULLS ON
+SET QUOTED_IDENTIFIER ON
+CREATE TABLE [dbo].[sys_all_views_table_vu_prepare](
+	[a] [int] NULL
+) ON [PRIMARY]
+
+GO
+
+SET ANSI_NULLS ON
+SET QUOTED_IDENTIFIER ON
+CREATE VIEW sys_all_views_dep_view_vu_prepare AS
+SELECT name, type, with_check_option FROM sys.all_views where object_id = object_id('sys_all_views_select_vu_prepare')
+GO
+
+SET ANSI_NULLS ON
+SET QUOTED_IDENTIFIER ON
+CREATE VIEW sys_all_views_select_chk_option_vu_prepare AS
+SELECT * FROM sys_all_views_table_vu_prepare
+WITH CHECK OPTION
+GO
+
+SET ANSI_NULLS ON
+SET QUOTED_IDENTIFIER ON
+CREATE VIEW sys_all_views_select_vu_prepare AS
+SELECT * FROM sys_all_views_table_vu_prepare
+GO
+
+SET ANSI_NULLS ON
+SET QUOTED_IDENTIFIER ON
+CREATE VIEW sys_views_dep_vu_prepare_t1 AS select 1;
```

Comment:
This is not a valid syntax in TSQL. 
```
1> CREATE VIEW sys_views_dep_vu_prepare_t1 AS select 1;
2> go
Msg 4511, Level 16, State 1, Server EC2AMAZ-EULFEOJ, Procedure sys_views_dep_vu_prepare_t1, Line 1
Create View or Function failed because no column name was specified for column 1.
1> 
```

1. You can remove this test.
2. Why do we have 2 extra "GO" at the end?